hack/docs/internal/cronjob-tutorial/webhook_implementation.go

```diff
@@ -205,7 +205,7 @@ const webhookTestCreateDefaultingReplaceFragment = `It("Should apply defaults wh
 			obj.Spec.FailedJobsHistoryLimit = nil     // This should default to 1
 
 			By("calling the Default method to apply defaults")
-			defaulter.Default(ctx, obj)
+			_ = defaulter.Default(ctx, obj)
```

Comment:
Now, you need run `make generate-docs` to have the docs updated accordingly
If not that will fails in the ci.
I mean, if the result of the `make generaate-docs` is != of the docs then it will not pass.

Reply:
Just ran and amended the commit. thought this would be done automatically later by a bot or so 😅

Reply:
That is because your branch is outdated.
Can you please rebase it with the master branch and re-run?

Reply:
Thanks @camilamacedo86. I just rebased it now and re-ran the generate-docs task.